iamy/aws.go

```diff
@@ -196,6 +197,16 @@ func (a *AwsFetcher) marshalPolicyDescriptionAsync(policyArn string, target *str
 	}()
 }
 
+func (a *AwsFetcher) fetchPolicyTags(policyArn string, tags *map[string]string) {
+	log.Println("Fetching tags for", policyArn)
```

Comment:
Do you need a trailing space after the "for" ?

Reply:
Nah, go helpfully (sometimes not so much) adds a space between arguments when printing strings. Output looks like:

```DEBUG 2022/01/25 08:37:05 Fetching tags for arn:aws:iam::909551307430:policy/invoke-initiator-look-up
```

Reply:
Hooray for Go! (first time that's ever been said, I imagine).